Source/ASRunLoopQueue.mm

```diff
@@ -31,8 +31,6 @@ static void runLoopSourceCallback(void *info) {
 
 #pragma mark - ASDeallocQueue
 
-@interface ASDeallocQueueV1 : ASDeallocQueue
```

Comment:
Let's rename `ASDeallocQueueV2` to `ASDeallocQueue` instead of having the version at the end.